var/spack/repos/builtin/packages/python/package.py

```diff
@@ -58,7 +58,7 @@ class Python(AutotoolsPackage):
     # See http://bugs.python.org/issue29846
     variant('shared', default=sys.platform != 'darwin',
             description='Enable shared libraries')
-    variant('tk', default=False, description='Provide support for Tkinter')
+    variant('tk', default=True, description='Provide support for Tkinter')
```

Comment:
This should not be changed here (and is breaking the CI). We should rather request `python+tk` in the `deploy/` level configurations.